libbeat/processors/move_fields/docs/move_fields.asciidoc

```diff
@@ -0,0 +1,113 @@
+[[processor-move-fields]]
+=== Move fields
+
+++++
+<titleabbrev>move-fields</titleabbrev>
+++++
+
+The `move-fields` processor allow you to arrange your message fields from a object into another object.
```

Comment:
We use the word "event" to refer to the data a Beat is ingesting/processing. Some other bits of this documentation also need this updating from "message" to "event".
```suggestion
The `move-fields` processor allow you to rearrange your event fields from a object into another object.
```